app/views/launchers/_common_launch.html.erb

```diff
@@ -17,6 +17,9 @@
   <%= render :partial => "launchers/common_head", :locals => {:launch_item => @launch_item } %>
   <div class="modal-body app_body">
     <%= hidden_field_tag :id, @launch_item.name %>
+    <% if @launch_item.options.any? %>
```

Comment:
@rajthilakmca  We will not need the `oneclick` as hidden_tag since we have it in the cache.  Hence before we launch, we can figure it out.